### DISCUSSION TITLE
Possibly a stupid question

### DISCUSSION THREAD

**Discussion by egol** [aevyrie/bevy_mod_picking#140] [Q&A] [answered]
The readme states this as a feature of the plugin: "Mouse intersection coordinates in world space." With what code can I get the Vec3 coordinates of the mouse intersection in world space?

**Comment by aevyrie** [comment, marked as answer]
The plugin is built on top of `bevy_mod_raycast`, which provides that functionality. See here: https://github.com/aevyrie/bevy_mod_raycast/blob/0a5d1e3334285eaf85e97adc28b14cbb50ab1b48/examples/minimal.rs#L69-L78

Note, you will need to use `bevy_mod_picking`'s `PickingRaycastSet` type here, or: 
```rs
query: Query<&Intersection<PickingRaycastSet>>
```

**Reply by egol** [reply]
Thanks for the answer! I'm still getting used to the Rust language and this helped me get past a major roadblock with how it functions.
